package.json

```diff
@@ -8,77 +8,77 @@
-        "@wdio/mocha-framework": "8.12.1",
-        "@wdio/selenium-standalone-service": "8.14.0",
-        "@wdio/spec-reporter": "8.12.2",
-        "autoprefixer": "^10.4.5",
-        "axios": "1.6.1",
-        "bootstrap-sass": "^3.4.1",
-        "chrome-launcher": "0.15.1",
-        "chromedriver": "123.0.0",
-        "commander": "11.1.0",
-        "cssnano": "6.0.1",
-        "ejs": "^3.1.7",
-        "eslint": "8.46.0",
+        "@babel/traverse": "^7.24.7",
+        "@lodder/grunt-postcss": "^3.1.1",
+        "@types/puppeteer": "^7.0.4",
+        "@wdio/browserstack-service": "^8.38.2",
+        "@wdio/cli": "^8.38.2",
+        "@wdio/local-runner": "^8.38.2",
+        "@wdio/mocha-framework": "^8.38.2",
+        "@wdio/selenium-standalone-service": "^8.15.0",
+        "@wdio/spec-reporter": "^8.38.2",
+        "autoprefixer": "^10.4.19",
+        "axios": "^1.7.2",
+        "bootstrap-sass": "^3.4.3",
+        "chrome-launcher": "^1.1.1",
+        "chromedriver": "^126.0.0",
+        "commander": "^12.1.0",
+        "cssnano": "^7.0.2",
+        "ejs": "^3.1.10",
+        "eslint": "^9.4.0",
         "eslint-config-prettier": "^9.1.0",
-        "expect-webdriverio": "3.1.4",
-        "glob": "^7.1.7",
-        "grunt": "^1.5.3",
+        "expect-webdriverio": "^4.14.0",
+        "glob": "^10.4.1",
+        "grunt": "^1.6.1",
         "grunt-cli": "^1.4.3",
         "grunt-contrib-concat": "^2.1.0",
         "grunt-contrib-copy": "^1.0.0",
         "grunt-contrib-watch": "^1.1.0",
         "grunt-regex-replace": "^0.4.0",
-        "grunt-shell": "4.0.0",
-        "grunt-stylelint": "^0.19.0",
-        "husky": "8.0.3",
+        "grunt-shell": "^4.0.0",
+        "grunt-stylelint": "^0.20.0",
+        "husky": "^9.0.11",
         "jasmine-ajax": "^4.0.0",
         "jasmine-core": "^5.1.2",
         "jasmine-jquery": "^2.1.1",
-        "karma": "6.4.3",
-        "karma-brief-reporter": "^0.2.1",
-        "karma-chrome-launcher": "^3.1.0",
-        "karma-coverage": "^2.0.3",
+        "jquery-migrate": "^1.4.1",
+        "karma": "^6.4.3",
+        "karma-brief-reporter": "^0.2.2",
+        "karma-chrome-launcher": "^3.2.0",
+        "karma-coverage": "^2.2.1",
         "karma-es6-shim": "^1.0.0",
-        "karma-firefox-launcher": "^2.1.1",
+        "karma-firefox-launcher": "^2.1.3",
         "karma-jasmine": "^5.1.0",
-        "karma-jasmine-html-reporter": "2.1.0",
+        "karma-jasmine-html-reporter": "^2.1.0",
         "karma-jasmine-matchers": "^5.0.0",
-        "karma-json-result-reporter": "^1.0.0",
         "karma-mocha-reporter": "^2.2.5",
         "karma-requirejs": "^1.1.0",
         "karma-safari-launcher": "^1.0.0",
-        "lint-staged": "^12.3.6",
+        "lint-staged": "^15.2.6",
         "load-grunt-tasks": "^5.1.0",
-        "msw": "0.35.0",
-        "postcss": "8.4.33",
-        "postcss-cli": "10.0.0",
-        "postcss-discard-comments": "6.0.0",
-        "postcss-scss": "^4.0.6",
+        "msw": "^2.3.1",
```

Comment:
We can't update that one yet until we rework how it gets used, as versions after 0.35.0 don't provide AMD files. Ideally, we use something different and remove it or rework the tests that make use of it entirely. It's not used in production things.

Reply:
Yeah, I found that out the hard way. 😖